client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -97,8 +97,9 @@ export default class Kudo extends React.Component {
         Save
       </button>
 
+    const recipientDisplay = map(this.props.kudo.receivers, 'name').join(', ')
     return <div className="kudo">
```

Comment:
It's more common for the `render` return format to be like this, right?:
```
return (
   <div>
      <img />
   </div> 
)
```

I noticed there's a mix of both styles throughout. Maybe we should do some code cleanup so the style is consistent? (I don't mind doing it!)

Reply:
Good call, I'll make this cleaner!  Good idea to cleanup as you find stuff.